unit-tests/dds/test-librs-intrinsics.py

```diff
@@ -151,8 +165,30 @@ def close_server( instance ):
 
         remote.run( 'close_server( instance )' )
         dev = None
-    #
-    #############################################################################################
+
+    with test.closure( 'Scaled intrinsics' ):
+        remote.run( 'instance = broadcast_device( build_scaled_intrinsics_server(), d435i.device_info )' )
+        dev = rs.wait_for_devices( context, rs.only_sw_devices, n=1. )
+        sensors = dev.query_sensors()
+        test.check_equal( len(sensors), 1 )
+        color = sensors[0]
+        profiles = color.profiles
+        resolutions = set()
+        for p in profiles:
+            vp = p.as_video_stream_profile()
+            if not vp:
+                continue
+            if vp.fps() != 30:
+                continue
+            if vp.format() != rs.format.yuyv:
+                continue
+            res = ( vp.width(), vp.height() )
+            if res not in resolutions:
+                log.out( f'resolution: {res} -> {vp.get_intrinsics()}' )  # throws if there's no intrinsics
```

Comment:
Can we also `check` expected intrinsic for resolutions other than 1920x1080, to verify scaling?

Reply:
I don't know what the right intrinsics are, and have no reference. They looked right when I did manual checking.
We should double-check that the algo is right. Maybe we can unit-test the algo in a separate test...